include/jemalloc/jemalloc_typedefs.h.in

```diff
@@ -75,3 +75,5 @@ struct extent_hooks_s {
 	extent_split_t		*split;
 	extent_merge_t		*merge;
 };
+
+typedef void (*prof_backtrace_hook_t)(void **, unsigned *, unsigned);
```

Comment:
This will make it publicly available in the upcoming 5.3 release, even if we don't include it in the manual. How about we keep it internal, like `/include/jemalloc/internal/hook.h`? That way people will have to dig into an internal header in order to use it, making it more experimental.

Reply:
Makes sense, I should have put it in `internal`.